pkgs/tools/networking/goflow2/default.nix

```
@@ -0,0 +1,26 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "goflow2";
+  version = "1.3.3";
+
+  src = fetchFromGitHub {
+    owner = "netsampler";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-YZuF3O1/Ycn2gFK9i1D/W8F16B6NEift5PYbv8yqUHk=";
+  };
+
```

this fixes `goflow2 -v`

```suggestion

  ldflags = [
    "-s"
    "-w"
    "-X=main.version=${version}"
  ];

```